model-optimizer/extensions/front/kaldi/replace_lstm_nonlinearity.py

```diff
@@ -27,12 +27,26 @@ def run_before(self):
         return [FullyConnectedDecomposer]
 
     def replace_op(self, graph: Graph, node: Node):
-        # split input to (i_part, f_part, c_part, o_part, ct_1)
         node_name = node.soft_get('name', node.id)
+        # check if we have dropout
+        input_port = node.in_port(0)
+        if node['use_dropout']:
```

Comment:
Maybe use node.soft_get('use_dropout')?

Reply:
All nodes lstm_nonlin have use_dropout attribute (Flase or True), so soft_get not needed.